source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/Rules/DateFormatMustBeUTCRule.cs

```diff
@@ -12,20 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
+using System.Globalization;
+using System.Text;
 using System.Text.RegularExpressions;
+using Energinet.DataHub.MeteringPoints.Domain.Extensions;
 using Energinet.DataHub.MeteringPoints.Domain.SeedWork;
+using NodaTime;
+using NodaTime.Text;
 
 namespace Energinet.DataHub.MeteringPoints.Domain.MeteringPoints.Rules
 {
     public class DateFormatMustBeUTCRule : IBusinessRule
     {
-        private const string FormatRegEx = @"\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2]\d|3[0-1])T23:00:00(.000)?Z$";
+        private const string FormatRegExSummer = @"\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2]\d|3[0-1])T23:00:00(.000)?Z$";
+        private const string FormatRegExWinter = @"\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2]\d|3[0-1])T22:00:00(.000)?Z$";
         private readonly string _date;
 
         public DateFormatMustBeUTCRule(string date)
         {
             _date = date;
-            IsBroken = !Regex.IsMatch(date, FormatRegEx);
+            var canParse = DateTime.TryParse(
+                date,
+                out var parseSuccess);
+
+            if (!canParse)
+            {
+                IsBroken = true;
+                return;
+            }
+
+            var info = TimeZoneInfo.FindSystemTimeZoneById("W. Europe Standard Time");
+            var tzi = info.IsDaylightSavingTime(parseSuccess);
+
+            if (tzi)
+            {
+                IsBroken = !Regex.IsMatch(date, FormatRegExSummer);
+            }
+            else
+            {
+                IsBroken = !Regex.IsMatch(date, FormatRegExWinter);
+            }
```

Comment:
nit: the only thing that changes in the if statement is the format, that could be the only element in the if statement then
```suggestion
            var format = isDaylightSavingTime ? FormatRegExSummer : FormatRegExWinter;
            IsBroken = !Regex.IsMatch(date, format);
```

Reply:
Has been changed. The if statement was for debugging purposes